server/index.js

```diff
@@ -42,15 +42,17 @@ io.on('connection', function(socket) {
 		io.emit('setDirection', data);
 	});
 });
+setInterval(function() {
+	b.render();
```

Comment:
This will fail. The server-side shouldn't be doing rendering. There is no canvas or document on the server.
Currently, the render function does 2 things:
1) compute positions & check for collisions (which is just math & coordinate/booleans calculations)
2) actually draw things on a canvas

These should be separated into 2 functions. The former will be called server-side; the latter will be called client-side upon receiving all positions and stuff